compute/compute-node.Dockerfile

```diff
@@ -1264,18 +1264,14 @@ RUN set -e \
     && make -j $(nproc) dist_man_MANS= \
     && make install dist_man_MANS=
 
-#########################################################################################
-#
-# Layers "postgres-exporter", "pgbouncer-exporter", and "sql-exporter"
-#
-#########################################################################################
-
-FROM quay.io/prometheuscommunity/postgres-exporter:v0.16.0 AS postgres-exporter
-FROM quay.io/prometheuscommunity/pgbouncer-exporter:v0.10.2 AS pgbouncer-exporter
-
-# Keep the version the same as in build-tools.Dockerfile and
-# test_runner/regress/test_compute_metrics.py.
-FROM burningalchemist/sql_exporter:0.17.0 AS sql-exporter
+FROM alpine:latest AS exporters
+RUN apk add --no-cache wget tar\
+    && wget -qO- https://github.com/prometheus-community/postgres_exporter/releases/download/v0.16.0/postgres_exporter-0.16.0.linux-amd64.tar.gz\
```

Comment:
This won't work on ARM, we probably can use `${TARGETARCH}` instead of the hardcoded `amd64` part (don't forget to add `ARG TARGETARCH` in this case)